Tribler/Core/Libtorrent/LibtorrentMgr.py

```diff
@@ -182,9 +197,8 @@ def create_session(self, hops=0):
             ltsession_settings['download_rate_limit'] = self.tribler_session.config.get_libtorrent_max_download_rate()
             ltsession.set_settings(ltsession_settings)
 
-        ltsession.add_dht_router('router.bittorrent.com', 6881)
-        ltsession.add_dht_router('router.utorrent.com', 6881)
-        ltsession.add_dht_router('router.bitcomet.com', 6881)
```

Comment:
The bitcomet router seems to have been retired. Atleast, this DNS name is no longer resolvable, and I can't find a new one in their domain.